svc/FetchService.ts

```diff
@@ -281,60 +270,59 @@ export class FetchService extends HoistService {
 
 /**
  * Standard options to pass through to fetch, with some additions.
- * [See MDN for available options]{@link https://developer.mozilla.org/en-US/docs/Web/API/Request}.
+ * See MDN for available options - {@link https://developer.mozilla.org/en-US/docs/Web/API/Request}.
  */
 export interface FetchOptions {
-    /** url for the request. Relative urls will be appended to XH.baseUrl.*/
+    /** URL for the request. Relative urls will be appended to XH.baseUrl. */
     url: string;
 
-    /** data to send in the request body (for POSTs/PUTs of JSON).*/
-    body?: object;
+    /** Data to send in the request body (for POSTs/PUTs of JSON).*/
+    body?: PlainObject;
 
     /**
-     * parameters to encode and append as a query string, or send with the request body
+     * Parameters to encode and append as a query string, or send with the request body
      * (for POSTs/PUTs sending form-url-encoded).
      */
-    params?: object;
+    params?: PlainObject;
 
     /**
-     * HTTP Request method to use for the request. If not specified,
-     * the method will be set to POST if there are params, otherwise GET.
+     * HTTP Request method to use for the request. If not specified, the method will be set to POST
+     * if there are params, otherwise GET.
      */
     method?: string;
 
     /**
-     *  Headers to send with this request. A Content-Type header will be set if not provided by
-     *  the caller directly or via one of the xxxJson convenience methods.
+     * Headers to send with this request. A Content-Type header will be set if not provided by
+     * the caller directly or via one of the xxxJson convenience methods.
      */
-    headers?: object;
+    headers?: PlainObject;
 
     /**
-     * ms to wait for response before rejecting with a timeout
-     * exception. Defaults to 30 seconds, but may be specified as null to specify no timeout.
+     * MS to wait for response before rejecting with a timeout exception. Defaults to 30 seconds,
+     * but may be specified as null to specify no timeout.
      */
-    timeout?: number|object;
+    timeout?: PromiseTimeoutSpec;
 
     /**
-     * optional metadata about the underlying request. Passed through
-     * for downstream processing by utils such as {@see ExceptionHandler}.
+     * Optional metadata about the underlying request. Passed through for downstream processing by
+     * utils such as {@link ExceptionHandler}.
      */
-    loadSpec?: object;
+    loadSpec?: PlainObject;
 
     /**
-     * options to pass to the underlying fetch request.
+     * Options to pass to the underlying fetch request.
      * @see https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/fetch
      */
-    fetchOpts?: object;
+    fetchOpts?: PlainObject;
```

Comment:
I wanted to use the `RequestInit` type suggested to me in some form by IJ as the parameter type for the native fetch, but it wasn't resolved.  Same with `Response`. Would like to learn more....